src/util.js

```diff
@@ -24,6 +24,20 @@ var Util = {
         total: 0,
         message: null
     },
+    
+    countOccurences: function(contentString, searchString) {
```

Comment:
nit: occurrences has two r's

(Disclaimer: I personally do this all the time; just pointing it out since i happened to google occurrence)